client/my-sites/media-library/title-item.jsx

```diff
@@ -0,0 +1,13 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+const TitleItem = props => {
+	/* eslint-disable wpcalypso/jsx-classname-namespace */
+	return (
+		<h6 className="section-nav-tab__title">{ props.children }</h6>
```

Comment:
Regarding the [CSS style guide](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#classes), shouldn't this class be something like `className=media-library__title-item`? That'd make unnecessary disabling the lint rule.